### ISSUE TITLE
Command_RotateToAzimuth giving negative value

### ISSUE THREAD

**Issue by erSuyognp** [SimDynamX/STU_Lunar_SAR#1]
I have implemented this function in MM 
![image](https://github.com/user-attachments/assets/c77f73b6-b9ce-4c93-9b27-a81c4ca41a2a)
But while I see in the log file, the given angle shows -3.1 instead of 3.1. Because of that it never says the command is complete and is in Pending forever.

Log file:
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: OnScreenLogMessage: LTV2 Behavior: Received TurnToAzimuth command; rotating to azimuth = -3.1 degrees.
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: OnScreenLogMessage: MoveToCoord command complete.
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}
2024-11-16T17:58:23 [INFO]----[PySys_MissionManager]: LTV2 task status: {'Pending': ['M2'], 'Completed': ['M1'], 'Failed': []}

**Comment by connorjak** [comment]
Good catch! I made a mistake in the SetupAllCommands function; it included `TurnToAzimuth` (the SurfaceMover function) instead of `RotateToAzimuth` (the command type).

Also on the RotateToAzimuth command, we had inserted a negative sign in its command-creation implementation in STU_Common.py to correct the direction of the rotation. The SurfaceMover code that is used inside EntityBehavior has a backwards azimuth direction. We didn't correct the sign again in the log message yet.
